adr/020-azure-alerts.md

```diff
@@ -14,12 +14,31 @@ Accepted.
 
 As part of our CI/CD infrastructure, we need notifications when failures occur.
 
-Current alert is configure to be:
-- [Azure Log Search Alerts](https://learn.microsoft.com/en-us/azure/azure-monitor/alerts/alerts-types#log-alerts) for HikariCP Connection failures
-  - Trigger on any logged failures with database connections as Azure metrics weren't capturing the issues found in our logs.
-  - Stateful (Auto-mitigation), to keep alerts in a `fired` status until resolved and for less noise on frequent/duplicate alerts.
-  - Configured to a Slack channel email until our Pagerduty is set up.
+To ensure rapid response to application failures within our CI/CD infrastructure, we require real-time notifications for critical issues. The current alert setup focuses on:
 
+- **Type:** [Azure Log Search Alerts](https://learn.microsoft.com/en-us/azure/azure-monitor/alerts/alerts-types#log-alerts) for HikariCP connection failures.
+
+
+- **Trigger:** Any logged failures with database connections.
+
+
+- **Configuration:** Alerts are stateful (auto-mitigation); set to `fired` status to reduce noise from frequent or duplicate alerts.
+
+
+- **Notification:** Alerts sent to a Slack channel via email until PagerDuty is operational.
+
+## Consequences:
```

Comment:
Change this to Impact?  Also, there is no need for a `:` because this is a heading and already indicates that it applies to what is to come.

```suggestion
## Impact
```